util/dvsim/FlowCfg.py

```diff
@@ -443,6 +447,35 @@ def write_results_html(self, filename, text_md):
         # Link the `/latest` folder with the latest reports.
         mk_symlink(result_path, symlink_path)
 
+    def write_results_hjson(self, test_hjson):
+        ''' Merge and write test_hjson to results.hjson.
+
+        First check if there are existing `results.hjson` file. If exists, append the current
+        results to the list of previous results hjson. Remove the oldest timestamps if
+        `results.hjson` file contains more than 120 entries.
+        '''
+        result_path = os.path.join(self.results_path, "results.hjson")
+        symlink_path = os.path.join(self.symlink_path, "results.hjson")
+
+        # If `results.hjson` exists, append the current hjson result and remove
+        # older entries if more than 120 entries are in the file.
+        if (os.path.isfile(symlink_path)):
```

Comment:
More stray parens! Also, I'd suggest `os.path.exists` here: I think the behaviour if this turns out to be a directory will be less confusing that way.